integration_tests/steps/power_tests.py

```diff
@@ -44,8 +44,6 @@ def test_baseline_peak_energy_stats(context):
                                                'peak_percentage': 50.0,
                                                'baseline_peak_energy_kWh': 0.4},
                                     'export': {'peak_energy_kWh': 0.0}},
-                        'House 1 2': {'import': {'peak_energy_kWh': 0.0, 'capacity_kWh': 2.0},
-                                      'export': {'peak_energy_kWh': 0.0, 'capacity_kWh': 2.0}},
```

Comment:
continue from https://github.com/gridsingularity/d3a/pull/880/files#r500194689 : 
In order to have something like this here?